addons/product/models/product.py

```diff
@@ -338,6 +338,8 @@ def create(self, vals_list):
             # When a unique variant is created from tmpl then the standard price is set by _set_standard_price
             if not (self.env.context.get('create_from_tmpl') and len(product.product_tmpl_id.product_variant_ids) == 1):
                 product._set_standard_price(vals.get('standard_price') or 0.0)
+        self.invalidate_cache()
+        self.clear_caches()
```

Comment:
Why do you invalidate the cache here? Is this the actual record's cache or the overall cache? If the global cache is invalidated here, I'm pretty sure you are introducing others performance issue in others contexts.  (same comment applies for 'write' and 'unlink'